corehq/apps/user_importer/importer.py

```diff
@@ -658,15 +658,15 @@ def _process_web_user(self):
                 if cv["tableau_groups"] is not None:
                     groups_list = cv["tableau_groups"].split(',')
                     tableau_group_ids = get_tableau_group_ids_by_names(groups_list, self.domain)
+                profile = None
+                if cv["profile_name"]:
+                    profile = self.domain_info.profiles_by_name[cv["profile_name"]]
```

Comment:
Fair points - adding profiles to the invite workflow (and bulk upload) was work we did recently. This change was something I missed in a previous [PR](https://github.com/dimagi/commcare-hq/pull/34466). I included it here because since it's related.
My concern with not getting this fixed (either here or I can create a separate PR) is that the confirmed account does not have its profile set even though the uploading users expect it to be. I'll test it locally and staging to make sure it's working fine, does that alleviate concerns?